app/views/kaminari/_last_page.html.haml

```diff
@@ -0,0 +1,9 @@
+-#  Link to the "Last" page
```

Comment:
7 consecutive comments can be merged into one<br>Comment should have a space after the `#`